pkg/telemetryquerylanguage/tql/README.md

```diff
@@ -129,6 +129,10 @@ Getters allow for reading the following types of data. See the respective sectio
 
 It is possible to update the Value in a telemetry field using a Setter. For read and write access, the `GetSetter` interface extends both interfaces.
 
+## Logging inside a TQL function
+
+To emit logs inside a TQL function, declare an argument with a type of `Logger` anywhere in the parameter list within the function signature. The TQL will then inject a logger instance provided by the component that can be used to emit logs.
```

Comment:
Technically a Logger (nor anything else) still can't come after a variadic slice

Reply:
Good catch. I've removed the wording specifying where to place the parameter so that the guidance for variadic slices still applies.